modules/internal/BytesStringCommon.chpl

```diff
@@ -25,6 +25,27 @@ module BytesStringCommon {
   private use ByteBufferHelpers;
   private use String.NVStringFactory;
 
+  extern const CHPL_SHORT_STRING_SIZE : c_int;
+
+  extern record chpl__inPlaceBuffer {};
+
+  // Signal to the Chapel compiler that the actual argument may be modified.
+  pragma "fn synchronization free"
```

Comment:
Did you mean to drop the "no doc" here? It also got dropped from the `extern`s and `__serializeHelper`

Reply:
Yep, and I should drop the one from the next one, as well.

The reason for that is this is a module for which we don't generate any documentation. I see that there are some `no doc`s in this module, but I think they just remained there when I moved the functions from the String module.

(4-5 releases ago, we didn't have the `bytes` type and just `string`. To create the `bytes` type, I created this common module that provides common implementations for the two types. Initially everything was just moved directly from the string module)